src/Symfony/Component/HttpKernel/EventListener/ProfilerListener.php

```diff
@@ -94,9 +97,13 @@ public function onKernelResponse(FilterResponseEvent $event)
         $this->profiles[$request] = $profile;
 
         $this->parents[$request] = $this->requestStack->getParentRequest();
+
+        if ($this->profileStack instanceof ProfileStack) {
```

Comment:
should this be done also for subrequests, or only for master requests ?

Reply:
Thanks for your review. The exact same profiles that are currently added to the `$profiles` class property are added to the stack. So the behavior is not modified. So it add them depending of the "only master requests" arg.